Mathlib/Combinatorics/SimpleGraph/Connectivity/Connected.lean

```diff
@@ -301,6 +307,10 @@ lemma connected_top_iff : (completeGraph V).Connected ↔ Nonempty V := by simp
 @[deprecated (since := "2025-09-23")] alias bot_not_connected := not_connected_bot
 @[deprecated (since := "2025-09-23")] alias top_connected := connected_top
 
+lemma Connected.of_nonempty_subsingleton [Nonempty V] [Subsingleton V] (G : SimpleGraph V) :
```

Comment:
```suggestion
lemma Connected.of_subsingleton [Nonempty V] [Subsingleton V] (G : SimpleGraph V) :
```
is an acceptable lie, I would think, to preserve the series of lemmas with the same names